orange/src/main/java/com/hartwig/hmftools/orange/algo/linx/LinxReportableClusters.java

```diff
@@ -30,6 +30,14 @@ public static Set<Integer> findReportableClusters(@NotNull LinxData linx)
         svIds.stream().filter(svToCluster::containsKey).map(svToCluster::get).forEach(clusterIds::add);
 
         linx.somaticDrivers().stream().filter(x -> x.clusterId() >= 0).forEach(x -> clusterIds.add(x.clusterId()));
-        return clusterIds;
+
+        return clusterIds.stream().filter(x -> !isSimpleCluster(linx, x)).collect(toSet());
+    }
+
+    private static boolean isSimpleCluster(@NotNull LinxData linx, int clusterId)
```

Comment:
Minor but maybe rename to `isSimpleClusterAffectingNoExons`?

In my mind, a simple cluster is a cluster with no chains, but that is independent of whether exons are affected or not.